recipes/libcurl/all/conanfile.py

```diff
@@ -54,6 +54,11 @@ class LibcurlConan(ConanFile):
     def _is_mingw(self):
         return self.settings.os == "Windows" and self.settings.compiler != "Visual Studio"
 
+    @property
+    def _is_win_x_android(self):
+        return self.settings.os == "Android" and platform.system() == "Windows"
```

Comment:
```suggestion
        return self.settings.os == "Android" and tools.os_info.is_windows
```
(btw, there is also `tools.detected_os()` to stay in the conan framework)